src/window/index.ts

```diff
@@ -16,13 +16,7 @@ const console = Logging.background;
  */
 export const windowMapping: Record<string, number> = {};
 
-export function getWebRoot() {
-  if (/^(?:dev|test)/i.test(process.env.NODE_ENV || '')) {
-    return 'http://localhost:8888';
-  }
-
-  return 'app://.';
-}
+export const webRoot = 'app://';
```

Comment:
Given that we need to replace all calls to `getWebRoot` anyway, I don't know why we don't just inline it?

Reply:
I feel that we reference `webRoot` in enough locations that the constant makes sense. It was nice to only have to change one instance for the busted packaged builds but either way works for me 🙂.